crypto/ed25519/math/point.go

```diff
@@ -0,0 +1,92 @@
+package math
+
+import (
+	"crypto/subtle"
+
+	"chain/crypto/ed25519/internal/edwards25519"
+)
+
+type (
+	// Point is a point on the ed25519 curve.
+	Point edwards25519.ExtendedGroupElement
+
+	// EncodedPoint is a serialized representation of a point.
+	EncodedPoint [32]byte
```

Comment:
For what it's worth, curve25519-dalek has named types for everything (calling this one `CompressedEdwardsY`):

https://github.com/isislovecruft/curve25519-dalek/blob/master/src/curve.rs#L109